[DT-1451] Add examples in python in README (#425)
cognitedata/cdp-spark-datasource: README.md

| `subtrees` | `ingest` | Controls what should happen with subtrees without a root node in the input data. `ingest` says they will be processed and loaded into CDF, `ignore` will ignore all of them and `error` will stop the execution and raise an error (nothing will be ingested). |
| `batchSize`                | 1000    | The number of assets to write per API call.                                                                       |

### Setup for Python

> You may want to set up a jupyter notebook with `pySpark` running.
> 
> * Download spark version `2.4.5` [here](https://www.apache.org/dyn/closer.lua/spark/spark-2.4.5/spark-2.4.5-bin-hadoop2.7.tgz)
> * Follow the instuctions given [here](https://www.sicara.ai/blog/2017-05-02-get-started-pyspark-jupyter-notebook-3-minutes) except that your spark version will be different than what is written over there.
> * Start your jupyter notebook with the following command instead of simple `pyspark` (as written in the above link):

Reviewer: ```suggestion
You may want to set up a jupyter notebook with `pySpark` running.

* Download spark version `2.4.5` [here](https://www.apache.org/dyn/closer.lua/spark/spark-2.4.5/spark-2.4.5-bin-hadoop2.7.tgz)
* Follow the instructions given [here](https://www.sicara.ai/blog/2017-05-02-get-started-pyspark-jupyter-notebook-3-minutes), except that your Spark version will be `2.4.5`.
* Start your Jupyter notebook with the following command (instead of `pyspark` as in the link above):
```

```
pyspark --packages com.cognite.spark.datasource:cdf-spark-datasource_2.11:1.2.18
```

### Example (Scala)
```scala

> 

Reviewer: ```suggestion
```

  .save()
```

### Example (Python)
```python

> 

Reviewer: Please add some comments explaining to the user what's being done and why, like we do for Scala.

> 

Reviewer: ```suggestion
```

  .insertInto("assets")
```

```python

> 

Reviewer: ```suggestion
```

  .save()
```

```python

> 

Reviewer: ```suggestion
```

s"and aggregation = 'min' and granularity = '1d'")
```

```python

> 

Reviewer: ```suggestion
```

df.show()
```

```python

> 

Reviewer: ```suggestion
```

spark.sql("""insert into tablename values ("key", "values")""")
```

```python

> 

Reviewer: ```suggestion
```
